client/python/gradio_client/client.py

```diff
@@ -453,11 +453,7 @@ def predict(
             client.predict(5, "add", 4, api_name="/predict")
             >> 9.0
         """
-        inferred_fn_index = self._infer_fn_index(api_name, fn_index)
-        if self.endpoints[inferred_fn_index].is_continuous:
```

Comment:
Don't we still need this when the Client connects to Spaces running older versions of Gradio?